.github/how-to-write-a-good-pull-request.md

```diff
@@ -0,0 +1,12 @@
+# How to write a good pull request
+
+- Make it small
+- Do only one thing
+- Avoid re-formatting
+- Make sure the code builds
+- Make sure all tests pass
+- Add tests
+- Write useful descriptions and titles
+- Keep your branch history clean
```

Comment:
Not everyone may have the same understand of what this means. Can we be more specific?